### DISCUSSION TITLE
card组件，纯文字模式，文字少时，怎样次才能通栏呢

### DISCUSSION THREAD

**Discussion by lambda332** [TaleLin/lin-ui#1386] [求助]
card组件，纯文字模式，文字少时，怎样次才能通栏呢。
![image](https://user-images.githubusercontent.com/8685340/117266082-76b01000-ae87-11eb-98c4-8b26635136b0.png)

**Comment by lambda332** [comment]
修改card源码中wxss文件中card-container-ful为card-container-full{width:750rpx}.解决了该问题，在文字长度较短情况下，width:100%并没有通栏，而是自适应了文字长度。

**Reply by aQingYun** [reply]
嗯 这个缺失是组件设计得问题 欢迎提一个pr来修改一下👏。
